src/assets/locales/es.json

```diff
@@ -7,8 +7,7 @@
   "avatar-page.select": "Seleccionar",
```

Comment:
You can ignore the language files. These are just formatting updates from my attempt to add translations, which have been removed